src/main/java/de/dennisguse/opentracks/adapters/CustomLayoutFieldType.java

```diff
@@ -1,8 +1,9 @@
 package de.dennisguse.opentracks.adapters;
 
 enum CustomLayoutFieldType {
-    SHORT(0),
-    WIDE(1);
+    SHORT(100),
+    WIDE(200),
+    CLOCK(1);
```

Comment:
Feels like clock is different than short and wide.
Would a separate Enum be better in the long run?

Reply:
Okay, let me give it a spin